client/src/components/flashcard/flashcardModal.jsx

```diff
@@ -74,6 +89,13 @@ const FlashCardModal = (props) => {
             </Button>
             </div>
           </AddModuleModal>
+          <DeleteModal 
+            visible={deleteVisible} 
+            setVisible={setDeleteVisible} 
+            object={"module"}
+            moduleId={id}
+            setId={setId}
+          />
         </Card>
         {/* 
                         Self-note: Passing key below was critical to re-render each module from the beginning!
```

Comment:
Remember to delete these comments before the final showcase!

Reply:
yep, will do!